security-advisories/security-advisories.md

```diff
@@ -5,7 +5,13 @@
 This is a generated list of our released advisories. A more comprehensive view
 is available in the Security Center.
```

Comment:
This sentence was copied from the website where the list was dynamically generated.